src/query/graphite/native/aggregation_functions.go

```diff
@@ -279,7 +279,7 @@ func divideSeriesLists(ctx *common.Context, dividendSeriesList, divisorSeriesLis
 
 // aggregate takes a list of series and returns a new series containing the
 // value aggregated across the series at each datapoint using the specified function.
-// This function can be used with aggregation functionsL average (or avg), avg_zero,
+// This function can be used with aggregation functions average (or avg), avg_zero,
```

Comment:
Nice, how did you even find this heh

Reply:
I think I had touched this file by accident and linter had gone off, then I reverted but the fix stayed. Good old linter hah.